### ISSUE TITLE
Retry workflow

### ISSUE THREAD

**Issue by GINAGC** [filetrust/cdr-plugin-folder-to-folder#200]
workflow which uses sub folders in hd2/data to keep track of the status of files:

- new
- processing
- error (which could be retried)
- failed (after multiple retries)
- not-supported

Retry workflow should retry these files X times and only after that mark them as failed.

**Comment by GINAGC** [comment]
@MaNe2020 and @samarthdd to test and document the retry workflow.

**Comment by MaNe2020** [comment]
@GINAGC at the moment this is the flow:
- Start regular processing of dataset (clear --> load --> start)
- Once regular processing is done, start processing again (just start)
- Follow the processing on Kibana Dashboard
- Go to Processing and check Processing status
![image](https://user-images.githubusercontent.com/70108899/117630777-d22a1880-b17b-11eb-885b-74426d836f8b.png)
![image](https://user-images.githubusercontent.com/70108899/117630982-0c93b580-b17c-11eb-83c4-1b3b6bd285f6.png)

What would be great is if we can have flag or bar or something that will indicate that `retry` mechanism is in progress

**Comment by GINAGC** [comment]
@MaNe2020 then please add the documentation above to docs. Create a new issue for the request on the flag/bar.

**Comment by MaNe2020** [comment]
#13 added to FAQ with details about retry workflow
issue created: https://github.com/filetrust/cdr-plugin-folder-to-folder/issues/321
